flake.nix

```diff
@@ -71,7 +70,7 @@
         }) self.nixosConfigurations;
       };
 
-      pipelineFile = serokell-nix.lib.pipeline.mkPipelineFile self;
+      pipelineFile = serokell-nix.lib.pipeline.mkPipelineFile (self // { nixArgs = ["--impure"]; });
```

Comment:
Why do we need impure here? :thinking:

Reply:
Otherwise checks fails due to impurity:(

Reply:
related: https://github.com/NixOS/nixpkgs/issues/179172

Reply:
Alright, now it's properly pure